### PR TITLE
Vickyyy sch pom 1

### PR DESCRIPTION
**Pull request by VickyyySch**
Changed regarding comments

### CODE REVIEW

File: src/test/java/pages/BasePage.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-public abstract class BasePage {
+public abstract class BasePage{
```
Comment (dzmitryrak) [inline src/test/java/pages/BasePage.java:6]: пробел перед фигурной скобкой

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ Validate the user details form (e.g., first name, last name, postal code).
+ Confirm the correct items are displayed on the checkout summary page.
+ Test the "CANCEL" and "FINISH" buttons during the checkout process.
+* Compatibility and Responsiveness:
+ Test the application's compatibility across different browsers (Chrome, Firefox, Safari, Edge).
+ Check responsiveness on various devices (desktop, mobile, tablet).
+* Error Handling and Edge Cases:
+ Test error scenarios (e.g., network failure, server errors).
+ Verify error messages are displayed appropriately.
+ Test boundary values for fields (e.g., maximum and minimum lengths for inputs).
```
Comment (dzmitryrak) [inline README.md:34]: не забывай про названия бранчей

File: src/test/java/pages/CartPage.java [hunk 3]
```diff
@@ -3,16 +3,16 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
-public class CartPage extends BasePage {
-
-    public CartPage(WebDriver driver) {
-        super(driver);
-    }
+public class CartPage extends BasePage{
```
Comment (dzmitryrak) [inline src/test/java/pages/CartPage.java:6]: везде пробел перед фигурной скобкой

File: src/test/java/tests/CartTest.java [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
-import org.openqa.selenium.WebElement;
 import org.testng.annotations.Test;
+
 import static org.testng.Assert.assertEquals;
 
-public class CartTest extends BaseTest {
+public class CartTest extends BaseTest{
 
-    @Test
+    @Test()
```
Comment (dzmitryrak) [inline src/test/java/tests/CartTest.java:9]: ТРЕБУЮ description

File: src/test/java/tests/CheckoutTest.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
+package tests;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class CheckoutTest extends BaseTest{
+
+    @Test(description = "e2e")
```
Comment (dzmitryrak) [inline src/test/java/tests/CheckoutTest.java:9]: а давай лучше назовем тест а не просто e2e

File: src/test/java/tests/LoginTest.java [hunk 6]
```diff
@@ -16,6 +18,7 @@ public void successfulLogin() {
                 "Products", "User is not logged in or wrong page");
     }
 
+    @Ignore
```
Comment (dzmitryrak) [inline src/test/java/tests/LoginTest.java:21]: Удали игноры и enable = false. Пускай все запускается

**Review by dzmitryrak — APPROVED** [review]
